pages/resources/[page].js

```diff
@@ -260,6 +257,7 @@ function Resources() {
                     </Button>
 
                     <Button
+                      onClick={() => handleReset(resetForm)}
```

Comment:
I know this one is not ideal, but `onReset` for some reason was fired every time a component would re-render, not only when pressing the reset button itself...

Reply:
One thing I bet we could do is instead leverage the `onReset` prop on the Form component. Note how we dont have an `onClick` on the submit button?

Reply:
yes, this is how it used to be bofore - "onReset prop on the Form component" but after updating Formik to v2, onReset is being called not only when you click the "Reset". For example, it's also triggered when you hit "Search" making the whole form reset...
Maybe I need to investigate it once more...

Reply:
Hmmm I'd definitely like a better understanding, and would appreciate any investigation.